keps/0004-add-testing-infrastructure.md

```diff
@@ -68,21 +80,48 @@ This is a two phase proposal.
 
 ### Tests
 
-Add testing packages to:
+KUDO will be tested by several different test suites that validate individual components, KUDO, and frameworks.
+
+#### Unit tests
+
+Unit tests test a single component and require no network connectivity. These should be fast, run on every pull request,
```

Comment:
> no network connectivity

We should be more specific and define this as either "not requiring the internet at runtime", or "not starting up API objects"

Reply:
Instead of "no network connectivity" I'll say "does not require a Kubernetes API"